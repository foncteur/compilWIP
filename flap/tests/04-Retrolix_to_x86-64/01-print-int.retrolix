globals ()
  l00: %rdi <- copy 73;
  l01: observe_int();
  l02: exit;
end