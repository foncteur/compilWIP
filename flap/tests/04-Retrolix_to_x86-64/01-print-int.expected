Process exited with status exited(0).
STDOUT:
73
STDERR:



